populus/cli/upgrade_cmd.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import click
+from .main import main
+
+from populus.api.upgrade import (
+    upgrade_configs,
+)
+
+from populus.config.versions import (
+    LATEST_VERSION,
+)
+
+
+@main.command('upgrade')
+@click.pass_context
+@click.option(
+    '-t',
+    '--to-version',
+    'to_version',
+    default=LATEST_VERSION,
+)
+def upgrade_cmd(ctx, to_version):
+    """
+    Generate project layout with an example contract.
```

Comment:
This looks like a copy/paste error.  Update docstring since click will use it for the `help` text.